mne/_digitization/_utils.py

```diff
@@ -119,6 +119,10 @@ def _read_dig_points(fname, comments='%', unit='auto'):
 
     _, ext = op.splitext(fname)
     if ext == '.elp' or ext == '.hsp':
+        # XXX: This should be dead code, but is deeply buried in
+        #      read_dig_montage.
+        # raise RuntimeError('if you are reading isotrak files please use'
+        #                    ' read_dig_polhemus_isotrak')
```

Comment:
So we just leave it for now and remove it via deprecation? If so please add a variant of `deprecat*` here, as `git grep deprec` is a very useful tool for finding deprecation-related things.